olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -832,23 +832,23 @@ spec:
                       - name: KUBERNETES_MIN_VERSION
                         value: "v1.0.0"
                       - name: "IMAGE_queue-proxy"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-queue"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-queue:knative-v1.8"
                       - name: "IMAGE_activator"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-activator"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-activator:knative-v1.8"
                       - name: "IMAGE_autoscaler"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-autoscaler"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-autoscaler:knative-v1.8"
                       - name: "IMAGE_autoscaler-hpa"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-autoscaler-hpa"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-autoscaler-hpa:knative-v1.8"
                       - name: "IMAGE_controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-controller"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-controller:knative-v1.8"
                       - name: "IMAGE_webhook__webhook"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-webhook"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-webhook:knative-v1.8"
                       - name: "IMAGE_domain-mapping"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-domain-mapping"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping:knative-v1.8"
                       - name: "IMAGE_domainmapping-webhook"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-domain-mapping-webhook"
-                      - name: "IMAGE_storage-version-migration-serving-serving-1.8.0__migrate"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:knative-serving-storage-version-migration"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-domain-mapping-webhook:knative-v1.8"
+                      - name: "IMAGE_storage-version-migration-serving-serving-1.8__migrate"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-storage-version-migration:knative-v1.8"
```

Comment:
the new images are already live:

```
podman pull registry.ci.openshift.org/openshift/knative-serving-storage-version-migration:knative-v1.8
```

worked on my machine

Reply:
Yes they came from Serving builds: https://github.com/openshift-knative/serving/commit/2bf387520e1d923cb418362a88268c5d2b506f0c